### PR TITLE
doc: update links to top level sections

### PR DESCRIPTION
**Pull request by greg-fer**
Reworked links that pointed to the removed getting started section. Changed the old links to appropriate new top level sections and edited their context. NCSDK-22321.

### CODE REVIEW

File: doc/nrf/templates/Integration_template_README.rst [hunk 1]
```diff
@@ -47,10 +47,10 @@ Integration prerequisites
...
 Before you start the |NCS| integration with AVSystem's Coiote IoT Device Management, make sure that the following prerequisites are completed:
 
-   * :ref:`Setup of nRF Connect SDK <getting_started>` environment.
-   * :ref:`Setup of nRF9160 DK <ug_nrf9160_gs>`.
-   * :ref:`Create an nRF Cloud account <creating_cloud_account>`.
-   * Create an account for `AVSystem Coiote Device Management <Coiote Device Management server_>`_.
+* :ref:`Installation of the nRF Connect SDK <installation>`.
+* :ref:`Setup of nRF9160 DK <ug_nrf9160_gs>`.
+* :ref:`Create an nRF Cloud account <creating_cloud_account>`.
+* Create an account for `AVSystem Coiote Device Management <Coiote Device Management server_>`_.
```
Comment (peknis) [inline doc/nrf/templates/Integration_template_README.rst:53]: Maybe similar verb form to all items, like install, set up, create.

**Review by peknis — COMMENTED** [review]
Just one minor nit.

File: doc/nrf/templates/Integration_template_README.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/templates/Integration_template_README.rst:53]: Fair point, thanks.

**Review by Avalei — APPROVED** [review]

File: doc/nrf/documentation/images/doc_structure_search_results.PNG
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/documentation/images/doc_structure_search_results.PNG:1]: When I search for something, I don't get highlights on the search results page anymore. I only get the highlight when I open one of the search result entries, hence the change of the image and the doc.

**Review by peknis — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**